content/docs/sidebar.json

```diff
@@ -86,7 +86,7 @@
       },
       {
         "slug": "data",
-        "label": "Working with data",
+        "label": "Working with Data",
```

Comment:
- [ ] For another ticket but we should capitalize titles everywhere, probably.
- [ ] Also we could consider matching Titles and URLs (assuming it helps SEO).